### DISCUSSION TITLE
syntax mono c# add for termux

### DISCUSSION THREAD

**Discussion by ExtinctDrift19** [termux/termux-packages#11315] [Q&A] [answered]
nano

**Comment by Grimler91** [comment, marked as answer]
I have no idea what is being asked/requested here. A single sentence is never a well written bug report/feature request/question

**Reply by zairullahdev** [reply]
Confusing

**Reply by zairullahdev** [reply]
Depend on the IDE, if it support C# or not

**Reply by ExtinctDrift19** [reply]
Support syntax mono c# add for termux nano

**Reply by sylirre** [reply]
That should be asked on Nano upstream since we do not implement features for packages.

**Reply by ExtinctDrift19** [reply]
command syntax C# nano

**Reply by licy183** [reply]
Do you mean add a `.nanorc` file and let the nano editor highlight some keywords of `.cs`? You could try to checkout https://github.com/loglob/csharp.nanorc and put the files into `$PREFIX/usr/share/nano`.

**Reply by ExtinctDrift19** [reply]
which file to choose csharp.nanorc or csharp.colorful.nanorc

**Reply by licy183** [reply]
I think both are ok. Choose one as you like.

**Reply by ExtinctDrift19** [reply]
include syntax csharp.nanorc package default C# mono
